### PR TITLE
Adding a prompt when unlocking all admin users.

### PR DESCRIPTION
**Pull request by robbieaverill**
Magerun pull-request check-list:

- [x] Pull request against develop branch (if not, just close and create a new one against it)
- [x] README.md reflects changes (if any)

Subject: (50 characters or less)

Fixes pull request #762 

Changes proposed in this pull request:

- Merge conflicts resolved with #810 changes
- Compatibility changes made between LockdownCommand and UnlockCommand for consistency
- Messages adjusted slightly in UnlockCommand messages from original command

### CODE REVIEW

File: src/N98/Magento/Command/Admin/User/LockdownCommand.php [hunk 1]
```diff
@@ -55,13 +55,13 @@ protected function execute(InputInterface $input, OutputInterface $output)
 
         /** @var $dialog \Symfony\Component\Console\Helper\DialogHelper */
         $dialog = $this->getHelperSet()->get('dialog');
-        $confirm = $dialog->ask(
+        $confirm = $dialog->askConfirmation(
             $output,
-            '<info><comment>Are you sure? This will lock all admin accounts! Type "yes" to continue...</comment></info>'
-            . PHP_EOL
+            sprintf('<question>Really lock %d all admin users?</question> <comment>[n]</comment>: ', count($userIds)),
```
Comment (ktomk) [inline src/N98/Magento/Command/Admin/User/LockdownCommand.php]: ... all %d

**Comment by ktomk** [comment]
@robbieaverill: Only found one wording issue, rest looks good!

File: src/N98/Magento/Command/Admin/User/LockdownCommand.php (hunk 1 above)
Comment (robbieaverill) [inline src/N98/Magento/Command/Admin/User/LockdownCommand.php]: Fixed.

**Comment by ktomk** [comment]
@robbieaverill: Thanks!

**Pull request merged**